velox/dwio/dwrf/common/Common.h

```diff
@@ -271,4 +275,114 @@ class StripeInformation {
   virtual uint64_t getNumberOfRows() const = 0;
 };
 
+static CompressionKind convertCompressionKind(
```

Comment:
can we make this static mapping using switch statement instead? is this intended to be used outside of this scope? if not let's put it in anonymous namespace, and move the definition to cpp file

Reply:
Done, this indeed should be placed in unnamed namespace.